.github/workflows/continuous-integration-workflow.yml

```diff
@@ -71,9 +71,9 @@ jobs:
     runs-on: ubuntu-latest
     if: needs.prerequisites.outputs.runjobs
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up gradle
-        uses: spring-io/spring-gradle-build-action@v1
+        uses: kirklin/spring-gradle-build-action@v1
```

Comment:
I don't think we want to use an action that is outside the Spring control, instead of doing this, we should open an issue in https://github.com/spring-io/spring-gradle-build-action in order to get a new version released with the deprecations replaced.

Please, let's keep `spring-io/spring-gradle-build-action@v1` for now.

Reply:
ok!I will sync the update to `spring-io/spring-gradle-build-action@v1`.